projects/fusion-ui/components/button/v4/button.component.html

```diff
@@ -0,0 +1,9 @@
+<button [class]="buttonClass" [class.fu-loading]="loading" [disabled]="disabled || loading">
+    <fusion-icon *ngIf="iconName" class="fu-button-icon" [name]="iconName"></fusion-icon>
```

Comment:
What happens if iconName and startIconName is pass to the component?

Reply:
For this component it not in inputs.